src/components/User/index.js

```diff
@@ -100,6 +118,50 @@ async function create(req, res, next) {
     }
 }
 
+async function login(req, res, next) {
+    try {
+        const { email, password } = req.body;
```

Comment:
before you get params you must validate body
better use value after validation 
like this 
 const { error, value } = UserValidation.login(req.body);
 const user = await UserService.findByEmail(value.email);